src/MOI_wrapper.jl

```diff
@@ -0,0 +1,399 @@
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIU = MOI.Utilities
+
+# indices
+const VI = MOI.VariableIndex
+const CI = MOI.ConstraintIndex
+# supported functions
+const SVF = MOI.SingleVariable
+const SAF = MOI.ScalarAffineFunction{Float64}
+# supported sets
+const Bounds = Union{MOI.EqualTo{Float64}, MOI.GreaterThan{Float64},
```

Comment:
Nit: Constants should be all upper-case (http://www.juliaopt.org/JuMP.jl/dev/style/#Naming-1), e.g., `BOUNDS` and `VAR_TYPES`.

Reply:
Yeah, I felt like this applies more to numerical constants etc., but I don't care enough myself and wish we had a working formatting tool.